bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/AbstractRetryingRpcListener.java

```diff
@@ -166,9 +166,10 @@ public void onClose(Status status, Metadata trailers) {
     }
 
     // Perform Retry
-    String channelId = trailers != null ? trailers.get(ChannelPool.CHANNEL_ID_KEY) : "";
-    LOG.info("Retrying failed call. Failure #%d, got: %s on channel %s",
-        status.getCause(), failedCount, status, channelId);
+    String channelId =
+        trailers != null ? trailers.get(ChannelPool.InstrumentedChannel.CHANNEL_ID_KEY) : "";
```

Comment:
Might be a good idea to hide the key extraction behind a function on the channel pool. Something like:

ChannelPool.extractIdentifier(trailers)